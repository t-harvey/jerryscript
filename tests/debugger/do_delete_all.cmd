break do_delete_all.js:17
b do_delete_all.js:18
b do_delete_all.js:21
list
delete all
next
c
