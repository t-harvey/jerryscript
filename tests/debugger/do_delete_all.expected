Connecting to: localhost:5001
Stopped at tests/debugger/do_delete_all.js:15
(jerry-debugger) break do_delete_all.js:17
Breakpoint 1 at tests/debugger/do_delete_all.js:17
(jerry-debugger) b do_delete_all.js:18
Breakpoint 2 at tests/debugger/do_delete_all.js:18
(jerry-debugger) b do_delete_all.js:21
Breakpoint 3 at tests/debugger/do_delete_all.js:21 (in delete_test() at line:20, col:1)
(jerry-debugger) list
1: tests/debugger/do_delete_all.js:17
2: tests/debugger/do_delete_all.js:18
3: tests/debugger/do_delete_all.js:21 (in delete_test() at line:20, col:1)
(jerry-debugger) delete all
(jerry-debugger) next
Stopped at tests/debugger/do_delete_all.js:16
(jerry-debugger) c
Connection closed.
